flake.nix

```diff
@@ -1,7 +1,7 @@
 {
   inputs = {
     nixpkgs.follows = "haskellNix/nixpkgs";
-    haskellNix.url = "github:input-output-hk/haskell.nix";
+    haskellNix.url = "github:ch1bo/haskell.nix?ref=enable-haddock-tests";
```

Comment:
A bit unfortunate you had to use your own handle. Perhaps it is time for a dedicated org? :)